browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
```

Comment:
Ah yeah, forgot about that case since I almost never write JSX! Totally makes sense

![tmyk](http://i.ytimg.com/vi/GD6qtc2_AQA/maxresdefault.jpg)